### DISCUSSION TITLE
Welcome to hylo-lang discussions!

### DISCUSSION THREAD

**Discussion by kyouko-taiga** [hylo-lang/hylo#720] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community and let anyone contribute to the design and development of the Hylo programming language. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded (we ask everyone to follow our [code of conduct](https://github.com/val-lang/val/blob/main/CODE_OF_CONDUCT.md)).

**Comment by spinicist** [comment]
Hi - I've listened to the recent ADSP podcasts with Sean Parent on Val. There was mention of a Slack Channel and Teams meetings. Are those open to all or did I misunderstand?

**Reply by dabrahams** [reply]
Theyʻre open ro all. Links are in The ADSP show notes on their website

**Reply by spinicist** [reply]
Thanks! I should have checked there🤦‍♂️

**Comment by dabrahams** [comment]
Now that I'm at a computer:

Hylo Teams Meeting
[Click here to join the meeting](https://teams.microsoft.com/l/meetup-join/19%3ameeting_YjZmOTJiMjUtNDZhNy00MTcxLWJjY2YtMTQ0ZGEzY2RkY2E4%40thread.v2/0?context=%7b%22Tid%22%3a%22fa7b1b5a-7b34-4387-94ae-d2c178decee1%22%2c%22Oid%22%3a%22a102b458-98db-4c5e-acad-cfa08a096ae3%22%7d)
Meeting ID: 298 158 296 273
Passcode: D2beKF
When: Tues/Thurs 12:30-1:00 PST
[Hylo Slack](https://join.slack.com/t/val-qs97696/shared_invite/zt-1z3dsblrq-y4qXfEE6wr6uMEJSN9uFyg)
